observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -88,6 +88,8 @@ syslog:: Forwards logs to an external syslog server.
 
 Each output type has its own configuration fields.
 
+include::modules/log6x-configuring-otlp-output.adoc[leveloffset=+1]
```

Comment:
Was this line and the "configuring otlp output" file added to this PR intentionally? It seems unrelated to the pico size and the release-notes PR also seems to include the same file.